src/main/java/project_manager_task_init_use_case/PMTaskInitInteractor.java

```diff
@@ -0,0 +1,37 @@
+package project_manager_task_init_use_case;
+
+import entity.CommonTask;
+import entity.CommonTaskFactory;
+import entity.CommonTaskFactoryInterface;
+import entity.Task;
+
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PMTaskInitInteractor implements PMTaskInitInputBoundary{
+
+    final PMTaskInitOutputBoundary presenter;
+
+    final CommonTaskFactory commonTaskFactory;
+
+    final PMTaskInitGateway taskInitGateway;
+
+    public PMTaskInitInteractor(PMTaskInitOutputBoundary presenter, CommonTaskFactory commonTaskFactory, PMTaskInitGateway gateway) {
```

Comment:
Don't use concrete CommonTaskFactory, use general TaskFactory

Reply:
Yes, I have already changed it